src/main.rs

```diff
@@ -1,3 +1,5 @@
+mod name_gen;
```

Comment:
This is maybe premature modularization, but it's fine here I think. Just don't go too far in the Java direction! We're allowed to do multiple things in one file!